lib/uo_cli.rb

```diff
@@ -188,66 +197,84 @@ def self.create_project_folder(dir_name)
-            end
+
+            # Download gemfile to user's local machine
             gem_path, gem_name = File.split(example_gem_file)
             example_gem_download = open(example_gem_file, {ssl_verify_mode: OpenSSL::SSL::VERIFY_NONE})
             IO.copy_stream(example_gem_download, File.join(dir_name, gem_name))
 
-            feature_path, feature_name = File.split(example_feature_file)
-            example_feature_download = open(example_feature_file, {ssl_verify_mode: OpenSSL::SSL::VERIFY_NONE})
-            IO.copy_stream(example_feature_download, File.join(dir_name, feature_name))
+            #if argument for creating an empty folder is not added
+            if empty_folder == false
+                
+                example_feature_file = "https://raw.githubusercontent.com/urbanopt/urbanopt-cli/master/example_files/example_project.json"
+            
+                remote_weather_files = [
+                    "https://raw.githubusercontent.com/urbanopt/urbanopt-cli/master/example_files/weather/USA_NY_Buffalo-Greater.Buffalo.Intl.AP.725280_TMY3.epw",
+                    "https://raw.githubusercontent.com/urbanopt/urbanopt-cli/master/example_files/weather/USA_NY_Buffalo-Greater.Buffalo.Intl.AP.725280_TMY3.ddy",
+                    "https://raw.githubusercontent.com/urbanopt/urbanopt-cli/master/example_files/weather/USA_NY_Buffalo-Greater.Buffalo.Intl.AP.725280_TMY3.stat",
+                ]
+
+                osm_files = [
+                    "https://raw.githubusercontent.com/urbanopt/urbanopt-cli/master/example_files/osm_building/7.osm",
+                    "https://raw.githubusercontent.com/urbanopt/urbanopt-cli/master/example_files/osm_building/8.osm",
+                    "https://raw.githubusercontent.com/urbanopt/urbanopt-cli/master/example_files/osm_building/9.osm"
+                ]
+
+                # Download weather file to user's local machine
+                remote_weather_files.each do |weather_file|
+                    weather_path, weather_name = File.split(weather_file)
+                    weather_download = open(weather_file, {ssl_verify_mode: OpenSSL::SSL::VERIFY_NONE})
+                    IO.copy_stream(weather_download, File.join(weather_dir_abs_path, weather_name))
+                end
+
+                osm_files.each do |osm_file|
+                    osm_path, osm_name = File.split(osm_file)
+                    osm_download = open(osm_file, {ssl_verify_mode: OpenSSL::SSL::VERIFY_NONE})
+                    IO.copy_stream(osm_download, File.join(osm_dir_abs_path, osm_name))
+                end
+
+                feature_path, feature_name = File.split(example_feature_file)
+                example_feature_download = open(example_feature_file, {ssl_verify_mode: OpenSSL::SSL::VERIFY_NONE})
+                IO.copy_stream(example_feature_download, File.join(dir_name, feature_name))
 
-            config_path, config_name = File.split(config_file)
```

Comment:
You aren't downloading the config file any longer, is that intentional?

Reply:
I think that's ok...they can copy their own in there.

Reply:
👍

Reply:
I deleted that by accident..its not creating it even in the project folder now. Should I add it back for the project folder?

Reply:
Yes, it should be in the regular project folder. Thanks!